config/locales/fr.yml

```diff
@@ -343,11 +351,16 @@ fr:
   page-type:
   page-types:
   pages: Pages
-  parent-page:
+  parent-page: 
   path:
-  register:
-  save:
-  sign-in-count:
+  profile:
+    certificate:
```

Comment:
@raakal @y2749zha Before this can be deployed to the production site we will have to translate the certificate text

Reply:
@rsmithlal Got it.  @y2749zha Could you send the text over to me via email in a word doc so that I can send it on to the translator?

Reply:
Can I send it to you in the body of the email?

Reply:
Yes. It just needs to be clear and without code formatting for easy translation